test/fixtures/hooks/multi/10-print_hello_kafo.rb

```diff
@@ -0,0 +1,13 @@
+boot('10-print_hello_kafo') do
+  app_option('--print-hello-kafo', :flag, 'Prints Hello Kafo when run is complete')
+end
+
+post('11-print_goodbye') do
+  puts "Goodbye"
+end
+
+post('10-print_hello_kafo') do
```

Comment:
This now doesn't test that it inherits the default name. You can leave this out and it should inherit `10-print_hello_kafo` already
```suggestion
post do
```